index.html

```diff
@@ -256,6 +256,217 @@ <h2 class="fs-m">Icon Button</h2>
+                            <img src="./assets/card.jpg" alt="Valorant" class=" horizontal-card-image">
+                            <div class="card-content">
+                                <h1 class="fs-m card-text-highlight">
+                                    VALORANT
+                                </h1>
+                                <p class="fs-s">
+                                    A 5v5 character-based tactical FPS where precise gunplay meets unique agent abilities.
+                                </p>
+                                <p class="fs-s">
+                                    <del>₹1499</del> ₹999
+                                    <span class="discount">34%</span>
+                                </p>
+                                <button class="btn btn-primary d-flex-center">
+                                    <img src="./assets/shopping-cart-solid.svg" alt="cart" class="btn-icon"> Add to Cart
+                                </button>
+                            </div>
+                        </div>
+                    </code>
+                </pre>
+
+                <!-- -------- Card with Text Overlay ---------  -->
+                <h2 class="fs-m">Card with Text Overlay</h2>
+                <div class="card text-overlay-card">
+                    <img src="./assets/card.jpg" alt="Valorant" class="card-image">
+                    <h1 class="fs-s card-text-highlight">
+                        VALORANT
+                    </h1>
+                    <p class="fs-s">
+                        A 5v5 character-based tactical FPS where precise gunplay meets unique agent abilities.
+                    </p>
+                    <p class="fs-s">
+                        <del>₹1499</del> ₹999 
+                        <span class="discount">34%</span>
+                    </p>
+                    <button class="btn btn-primary d-flex-center">
+                        <img src="./assets/shopping-cart-solid.svg" alt="cart" class="btn-icon"> Add to Cart
+                    </button>
+                    <p class="text-overlay discount">KillJoy JODD</p>
+                </div>
+
+                <pre class="code-container">
+                    <code class="html fs-s">
+<div class="card text-overlay-card">
+    <img src="./assets/card.jpg" alt="Valorant" class="card-image">
+    <h1 class="fs-s card-text-highlight">
+        VALORANT
+    </h1>
+    <p class="fs-s">
+        A 5v5 character-based tactical FPS where precise gunplay meets unique agent abilities.
+    </p>
+    <p class="fs-s">
+        <del>₹1499</del> ₹999 
+        <span class="discount">34%</span>
+    </p>
+    <button class="btn btn-primary d-flex-center">
+        <img src="./assets/shopping-cart-solid.svg" alt="cart" class="btn-icon"> Add to Cart
+    </button>
+    <p class="text-overlay discount">KillJoy JODD</p>
+</div>
+                    </code>
+                </pre>
+            </section>
```

Comment:
Check the indentation here, the rest all looks good.

Reply:
Made the changes.